ee/migrations/0008_null_def_properties_allowed.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.5 on 2022-02-08 23:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("ee", "0007_dashboard_permissions"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="enterpriseeventdefinition",
+            name="description",
+            field=models.CharField(blank=True, max_length=400, null=True),
+        ),
+        migrations.AlterField(
+            model_name="enterpriseeventdefinition",
+            name="verified",
+            field=models.BooleanField(blank=True, default=False, null=True),
```

Comment:
Allow null values for verified and description property so that api calls are backwards compatible (i.e., event definitions with undefined properties)